test/docker_tests/configurations/test_all_docker_composes.sh

```
@@ -0,0 +1,73 @@
+    docker volume prune -f 2>/dev/null || true
+
+    echo "" >> "$LOG_FILE"
+    echo "==========================================" >> "$LOG_FILE"
+    echo "" >> "$LOG_FILE"
+}
+find "$SCRIPT_DIR" -name "docker-compose*.yml" -type f -print0 | sort -z | while IFS= read -r -d '' file; do
+    extract_comments "$file"
+    run_test "$file"
+done
+    run_test "$file"
+done
```

Shellcheck (0.11.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)